theurgy.lic

```diff
@@ -776,7 +782,7 @@ class TheurgyActions
     return false unless holy_water?
     safe_walk_to @data['altar']['id']
     DRCI.get_item_safe(@water_holder, @theurgy_supply_container)
-    fput 'clean altar with holy water'
+    bput('clean altar with holy water','Roundtime','You are a bit too busy performing to do that')
```

Comment:
Do you even need the 'You are a bit too busy performing to do that' match? I thought bput automatically handled that as a match, too, even if unspecified?

Reply:
Nope!  I just got in a hurry removing the other lines.